src/parser/entities/types.ts

```diff
@@ -4,6 +4,8 @@ export * from "./circle";
 export * from "./dimension";
 export * from "./ellipse";
 export * from "./hatch";
+export * from './image/types'
+export type * from './image/consts'
```

Comment:
I realized that we export everything (including realized constants) via `types.ts`. I'll fix them later.